addon/hint/show-hint.js

```diff
@@ -379,12 +379,15 @@
     },
 
     scrollToActive: function() {
-      var node = this.hints.childNodes[this.selectedHint]
+      var margin = this.completion.options.scrollMargin || 0;
+      if (! Number.isInteger(margin)) margin = 0;
```

Comment:
We are still targeting IE9, which doesn't have `Number.isInteger`. The library as a whole doesn't do run-time type checking, so just dropping this line would be fine.

Reply:
done